src/web/app/pages-instructor/student-list/student-list-section-data.ts

```diff
@@ -20,3 +20,68 @@ export interface StudentListStudentData {
   team: string;
   photoUrl?: string;
 }
+
+/**
+ * Flattened data which contains details about a student and their section.
+ * The data is flattened to allow sorting of the table.
+ */
+export interface FlatStudentListData {
```

Comment:
`FlatStudentListData` is the internal data structure used by student-list.component so it should be hidden?

Reply:
Updated. Moved `FlatStudentListData` to the component itself.

Reply:
Same as SortBy and SortOrder?

Reply:
Oh yes. Updated